src/main/java/com/simsilica/lemur/component/TextEntryComponent.java

```diff
@@ -673,15 +699,23 @@ public void keyAction( TextEntryComponent source, KeyAction key ) {
     private static class CaratLeft implements KeyActionListener {
         @Override
         public void keyAction( TextEntryComponent source, KeyAction key ) {
-            source.model.left();
+            if (source.font.isRightToLeft()) {
+                source.model.right();
```

Comment:
I think this is a clever way to avoid making the document model aware of rightToLeft text but I wonder if we will regret it later.  I think probably I need to rename the DocumentModel methods for right/left to "forward", "back" to disambiguate direction... in which case this is 100% fine.